client/src/components/reader/Annotation/Popup/Wrapper.js

```diff
@@ -124,16 +148,47 @@ export default class AnnotationPopup extends Component {
     let direction = up;
     if (topCollisionPossible) direction = down;
     if (preferBottom && !bottomCollisionPossible) direction = down;
-    const naturalTopPosition = window.pageYOffset + rect.top - popupHeight;
+    const naturalTopPosition = window.pageYOffset + rect.top - 60;
```

Comment:
Could you leave a comment as to what the hard coded `60` represents so we know when we come back to it later?